### PR TITLE
Adding in a component to manage bookmarks on the new tab page. Fixes #34

### LINKED ISSUE #34: Make the homescreen actually change the URL when an icon is tapped

### PR DESCRIPTION
**Pull request by jonathanKingston**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
   <script src="node_modules/fxos-component/fxos-component.js"></script>
   <script src="lib/hope-tab.js"></script>
   <script src="lib/hope-tab-list.js"></script>
+  <script src="lib/hope-app-link.js"></script>
```
Comment (etiennesegonzac) [inline index.html]: is it needed here?

File: lib/hope-app-link.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      title: shadow.querySelector('.title'),
+      image: shadow.querySelector('img'),
+    };
+
+    this.apps = {
+      contacts: {
+        icon: 'contacts.png',
+        text: 'Contacts',
+        url: 'https://example.com'
+      },
```
Comment (etiennesegonzac) [inline lib/hope-app-link.js]: we can remove the completely fake ones :)

File: lib/hope-app-link.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+      'camera': {
+        icon: 'camera.png',
+        text: 'Camera',
+        url: 'https://example.com'
+      },
+      'new-york-times': {
+        icon: 'nyt.png',
+        text: 'New York Times',
+        url: 'https://example.com'
```
Comment (etiennesegonzac) [inline lib/hope-app-link.js]: nit: we can put a real domain here

File: lib/hope-app-link.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
+/*global fxosComponent */
```
Comment (etiennesegonzac) [inline lib/hope-app-link.js]: we should probably move this component into the `homescreen` dir.
long term we might have a completely separate prototyping track looking at the homescreen/new tab page, maybe a separate repo etc...

**Comment by etiennesegonzac** [comment]
Then we're good to go!

**Comment by jonathanKingston** [comment]
@etiennesegonzac made a separate commit for the unused app icons as we might want them back when that functionality is there.
I made the homepage app have it's own package.json and the main package.json install that when doing a normal install.

**Comment by etiennesegonzac** [comment]
awesome, thanks!

**Pull request merged**